tests/integration/build_invoke/test_build_invoke.py

```diff
@@ -0,0 +1,322 @@
+
+
+class BuildInvoke_nodejs12_x_cookiecutter_quick_start_s3(Base.BuildInvokeBase):
+    directory = "nodejs12.x/cookiecutter-quick-start-s3"
+
+    # if we want to check response json, we need to setup bucket for testing
+
+
+class BuildInvoke_nodejs12_x_cookiecutter_quick_start_sns(Base.BuildInvokeBase):
+    directory = "nodejs12.x/cookiecutter-quick-start-sns"
+
+
+class BuildInvoke_nodejs12_x_cookiecutter_quick_start_sqs(Base.BuildInvokeBase):
+    directory = "nodejs12.x/cookiecutter-quick-start-sqs"
+
+
+class BuildInvoke_nodejs12_x_cookiecutter_quick_start_web(Base.QuickStartWebBuildInvokeBase):
+    directory = "nodejs12.x/cookiecutter-quick-start-web"
+
+    # if we want to check response json, we need to setup dynamodb
+
+
+class BuildInvoke_nodejs10_x_cookiecutter_aws_sam_hello_nodejs(Base.SimpleHelloWorldBuildInvokeBase):
+    directory = "nodejs10.x/cookiecutter-aws-sam-hello-nodejs"
+
+
+class BuildInvoke_nodejs10_x_cookiecutter_quick_start_from_scratch(Base.BuildInvokeBase):
+    directory = "nodejs10.x/cookiecutter-quick-start-from-scratch"
+
+
+class BuildInvoke_nodejs10_x_cookiecutter_quick_start_cloudwatch_events(Base.BuildInvokeBase):
+    directory = "nodejs10.x/cookiecutter-quick-start-cloudwatch-events"
+
+
+class BuildInvoke_nodejs10_x_cookiecutter_quick_start_s3(Base.BuildInvokeBase):
+    directory = "nodejs10.x/cookiecutter-quick-start-s3"
+
+    # if we want to check response json, we need to setup bucket for testing
+
+
+class BuildInvoke_nodejs10_x_cookiecutter_quick_start_sns(Base.BuildInvokeBase):
+    directory = "nodejs10.x/cookiecutter-quick-start-sns"
+
+
+class BuildInvoke_nodejs10_x_cookiecutter_quick_start_sqs(Base.BuildInvokeBase):
+    directory = "nodejs10.x/cookiecutter-quick-start-sqs"
+
+
+class BuildInvoke_nodejs10_x_cookiecutter_quick_start_web(Base.QuickStartWebBuildInvokeBase):
+    directory = "nodejs10.x/cookiecutter-quick-start-web"
+
+    # if we want to check response json, we need to setup dynamodb
+
+
+class BuildInvoke_nodejs10_x_cookiecutter_aws_sam_step_functions_sample_app(Base.BuildInvokeBase):
+    directory = "nodejs10.x/cookiecutter-aws-sam-step-functions-sample-app"
+
+
+class BuildInvoke_python2_7_cookiecutter_aws_sam_hello_python(Base.SimpleHelloWorldBuildInvokeBase):
+    directory = "python2.7/cookiecutter-aws-sam-hello-python"
+
+
+class BuildInvoke_python2_7_cookiecutter_aws_sam_step_functions_sample_app(Base.BuildInvokeBase):
+    directory = "python2.7/cookiecutter-aws-sam-step-functions-sample-app"
+
+
+class BuildInvoke_python3_6_cookiecutter_aws_sam_hello_python(Base.SimpleHelloWorldBuildInvokeBase):
+    directory = "python3.6/cookiecutter-aws-sam-hello-python"
+
+
+class BuildInvoke_python3_6_cookiecutter_aws_sam_eventBridge_python(Base.EventBridgeHelloWorldBuildInvokeBase):
+    directory = "python3.6/cookiecutter-aws-sam-eventBridge-python"
+
+
+@skip("eventbridge schema app seems not be able to build")
```

Comment:
Lets investigate this.

Reply:
I always got this error and nothing I can find online. Looking if other people know about it

```
Build Failed
Running JavaGradleWorkflow:GradleBuild
Error: JavaGradleWorkflow:GradleBuild - Gradle Failed: /tmp/samcli/source/src/main/java/helloworld/App.java:9: error: package aws.ec2.EC2InstanceStateChangeNotification does not exist
import aws.ec2.EC2InstanceStateChangeNotification.AWSEvent;
                                                 ^
/tmp/samcli/source/src/main/java/helloworld/App.java:10: error: package aws.ec2.EC2InstanceStateChangeNotification does not exist
import aws.ec2.EC2InstanceStateChangeNotification.EC2InstanceStateChangeNotification;
                                                 ^
/tmp/samcli/source/src/main/java/helloworld/App.java:11: error: package aws.ec2.EC2InstanceStateChangeNotification.marshaller does not exist
import aws.ec2.EC2InstanceStateChangeNotification.marshaller.Marshaller;
                                                            ^
/tmp/samcli/source/src/main/java/helloworld/App.java:20: error: cannot find symbol
    private Object handleEvent(final AWSEvent<EC2InstanceStateChangeNotification> inputEvent, final Context context) {
```